### ISSUE TITLE
Support Request: Support storysource to Prettier v1.13.0

### ISSUE THREAD

**Issue by isoppp** [storybooks/storybook#3657]
### Request Support Prettier v1.13.0

Prittier default parser option changed  babylon to empty and if settting empty, ouput error.

See: https://prettier.io/blog/2018/05/23/1.13.0.html#don-t-default-to-the-javascript-parser-4528-by-duailibe

### Steps to reproduce

New create using storysource project and run.

Out this error:

```
Error: No parser and no file path given, couldn't infer a parser.
```

### platform and version

- Mac 10.12.6
- Node.js v8.9.4

- "@storybook/vue@^3.4.6"
- "@storybook/addon-storysource@^3.4.6"

### Where to start

https://github.com/storybooks/storybook/blob/master/addons/storysource/src/loader/default-options.js

I tried add `parser: 'babylon'` in default option. And it works.
But I do not know if this modification alone is enough.

Add:
prettier changed output from error to warning in 1.13.1

**Comment by golgistudio** [comment]
Referencing the related prettier issue.

https://github.com/prettier/prettier/issues/4567

**Comment by isoppp** [comment]
@igor-dv
I think this need fix. But there's no rush.
Because Prettier changed to warning, but will change again error in the future.

Now this is output:
> No parser and no filepath given, using 'babylon' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.   

But if [this commit](https://github.com/storybooks/storybook/commit/6b3604c72588028f5a6e97f737eedac4a4b7f109) is enough (locke Prettier version forever) , don't need it.

**Comment by Hypnosphi** [comment]
> But if this commit is enough

That was a temporary solution until your PR gets merged

**Comment by isoppp** [comment]
It was merged, I close this.

@Hypnosphi 
Thanks for reviewing many times :bow:

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.9`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.9)
